demos/image_inpainting_demo/python/inpainting.py

```diff
@@ -15,32 +15,36 @@
 
 
 class ImageInpainting:
-    def __init__(self, ie, model_path, device='CPU'):
-        model = ie.read_network(model_path, model_path.with_suffix('.bin'))
+    def __init__(self, core, model_path, device='CPU'):
+        model = core.read_model(model_path, model_path.with_suffix('.bin'))
 
-        assert len(model.input_info) == 2, "Expected 2 input blob"
-        assert len(model.outputs) == 1, "Expected 1 output blobs"
+        if len(model.inputs) != 2:
+            raise RuntimeError("The model expects 2 input layers")
+        if len(model.outputs) != 1:
+            raise RuntimeError("The model expects 1 output layer")
 
-        self._input_layer_names = sorted(model.input_info)
-        self._output_layer_name = next(iter(model.outputs))
+        self.image_input_layer, self.mask_input_layer = sorted([node.get_any_name() for node in model.inputs])
 
-        self._ie = ie
-        self._exec_model = self._ie.load_network(model, device)
+        compiled_model = core.compile_model(model, device)
+        self.infer_request = compiled_model.create_infer_request()
 
-        _, channels, input_height, input_width = model.input_info[self._input_layer_names[0]].input_data.shape
-        assert channels == 3, "Expected 3-channel input"
+        _, channels, input_height, input_width = model.input(self.image_input_layer).shape
+        if channels != 3:
+            raise RuntimeError("The model expects 3 channels for {} input layer".format(self.image_input_layer))
 
-        _, channels, mask_height, mask_width = model.input_info[self._input_layer_names[1]].input_data.shape
-        assert channels == 1, "Expected 1-channel input"
+        _, channels, mask_height, mask_width = model.input(self.mask_input_layer).shape
+        if channels != 1:
+            raise RuntimeError("The model expects 3 channels for {} input layer".format(self.mask_input_layer))
```

Comment:
mask is single channel input
```suggestion
            raise RuntimeError("The model expects 1 channels for {} input layer".format(self.mask_input_layer))
```

Reply:
Thanks, it was a typo